emergence_lib/src/organisms/organism_details.rs

```diff
@@ -0,0 +1,157 @@
+//! Detailed info about a given organism.
+
+use bevy::prelude::*;
+use bevy_ecs_tilemap::tiles::TilePos;
+
+use std::fmt::Display;
+
+use crate::{
+    cursor::CursorTilePos,
+    items::{Inventory, Recipe},
+    structures::crafting::{
+        ActiveRecipe, CraftTimer, CraftingState, InputInventory, OutputInventory,
+    },
+};
+
+use super::{
+    sessile::{fungi::Fungi, plants::Plant},
+    units::Ant,
+};
+
+/// The type of the organism, e.g. plant or fungus.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum OrganismType {
```

Comment:
This is going to get clobbered by whatever we end up doing for organism IDs, but that's fine.